src/OSPSuite.Infrastructure.Import/Core/DataSourceFileReaders/ExcelReader.cs

```diff
@@ -84,6 +84,21 @@ public bool MoveToNextRow()
          return true;
       }
 
+      public IEnumerable<string> RetrieveExcelSheets(bool excludeEmptySheets)
```

Comment:
should that have a default of true (excludeEmptySheetS)

Reply:
I agree, it is better with a default value. Done.